modnet/hyper_opt/fit_genetic.py

```diff
@@ -646,7 +660,7 @@ def run(
 
         else:
             ensemble = []
-            for m in models[ranking[:10]]:
+            for m in models[ranking[:refit]]:
```

Comment:
Why ? 10 is arbitrary sure... But now, refit=0 is throwing an error unfortunately...

Reply:
Ah, oops. Shouldn't it be just the first model though?